app/services/eligibility_service.rb

```diff
@@ -86,7 +86,7 @@ def five_past_years_of_nulls
   def future_days_of_nulls
     days = {}
 
-    7.times do |i|
+    1.times do |i|
```

Comment:
If this is only happening once, then can we just put the actual contents of the method? In which case, we could back that up to the variable declaration which is the only return, so we could explicitly define the return e.g.:
```
def future_days_of_nulls
  {
    @day.to_date => nil
  }
end
```

Reply:
yes. I was getting this out quickly, so did the quickest hack.
If we're only reading one day worth on the controller, there's no need to slow down the open fisca call with 7 days of computations.